index.html

```diff
@@ -137,7 +137,7 @@ <h2 class="subtitle2">Professional Art Printing Data</h2>
           <li class="lang-box2">bootstrap</li>
           <li class="lang-box2">Ruby</li>
         </ul>
-        <button class="full-width-btn">See Project</button>
+        <button class="full-width-btn see-more projectSeven">See Project</button>
       </div>
     </div>
```

Comment:
- [ ] Kindly populate the project section dynamically with a javascript code that will be looping through the project array.
> eg:

```
projects.forEach((project) => {
     section.append(...)
})
```